rules/aws_cloudtrail_rules/aws_cloudtrail_infrastructure_discovery.yml

```diff
@@ -0,0 +1,59 @@
+AnalysisType: rule
+Description: This detection looks for Describe* and List* events in AWS CloudTrail logs. If these events occur in a short period of time from the same ARN, it could constitute Cloud Infrastructure Discovery reconnaissance.
+DisplayName: AWS CloudTrail Infrastructure Discovery
+Enabled: false
+Filename: aws_cloudtrail_infrastructure_discovery.py
+Reports:
+    MITRE ATT&CK:
+        - TA0007:T1580
+Severity: Info
+Tests:
+    - ExpectedResult: true
+      Log:
+        awsregion: ap-northeast-2
```

Comment:
everything in here is cased in a way that is inconsistent with the data represented in the logs... while the `event` object does respond to this either way,  it would be my preference that we maintain fidelity between our test cases and the original log data.  it looks like in snowflake, the quoting used around the column names will determine whether or not this case-insensitive approach will work...